python/pyspark/conf.py

```diff
@@ -101,13 +101,31 @@ def __init__(self, loadDefaults=True, _jvm=None, _jconf=None):
             self._jconf = _jconf
         else:
             from pyspark.context import SparkContext
-            SparkContext._ensure_initialized()
```

Comment:
Nevermind, this wouldn't even be possible here.  I didn't realize that `SparkConf` can be created first, launching the jvm, and then configuration is added after.